admin-ui/app/types/react-helmet.d.ts

```diff
@@ -0,0 +1,20 @@
+declare module 'react-helmet' {
+  import { ComponentType, HelmetData } from 'react-helmet'
+
+  interface HelmetProps {
+    title?: string
+    titleTemplate?: string
+    defaultTitle?: string
+    meta?: Array<{ name?: string; property?: string; content: string; httpEquiv?: string }>
+    link?: Array<{ rel?: string; href?: string; type?: string }>
+    script?: Array<{ src?: string; type?: string; innerHTML?: string }>
+    style?: Array<{ type?: string; cssText?: string }>
+    htmlAttributes?: { [key: string]: any }
+    bodyAttributes?: { [key: string]: any }
+    children?: React.ReactNode
```

Comment:
Reference ReactNode instead of React.ReactNode.

Line 14 uses `React.ReactNode` but the React namespace is not imported. After fixing the import on line 2, use `ReactNode` directly.



Apply this diff:

```diff
-    children?: React.ReactNode
+    children?: ReactNode
```